commercetools-models/src/test/java/io/sphere/sdk/states/commands/StateUpdateCommandIntegrationTest.java

```diff
@@ -92,4 +95,39 @@ public void changeType() {
             return updatedState;
         });
     }
+
+    @Test
+    public void setRoles() throws Exception {
+        withUpdateableState(client(), builder -> builder.type(StateType.REVIEW_STATE), state -> {
+            final Set<StateRole> roles = asSet(REVIEW_INCLUDED_IN_STATISTICS);
+            final StateUpdateCommand command = StateUpdateCommand.of(state, SetRoles.of(roles));
+            final State updatedState = client().executeBlocking(command);
+            assertThat(updatedState.getRoles()).isEqualTo(roles);
+            return updatedState;
+        });
+    }
+
+    @Test
+    public void addRoles() throws Exception {
```

Comment:
In this test I would have preferred that `AddRoles` is used with a state that has already roles otherwise `SetRoles` could have been used.